### PR TITLE
Data Persistence and Management System

### PR DESCRIPTION
**Pull request by BakJakub**

### CODE REVIEW

File: Frameworks/NetworkService/Sources/NetworkService/Logging/NetworkingLogger.swift [hunk 1]
```diff
@@ -21,15 +21,15 @@ class NetworkingLogger {
...
+	func log(response: URLResponse, data _: Data) {
 		guard logLevel != .off else {
 			return
 		}
 		if let response = response as? HTTPURLResponse {
 			logStatusCodeAndURL(response)
 		}
 		if logLevel == .debug {
-			print(String(decoding: data, as: UTF8.self))
+			// print(String(decoding: data, as: UTF8.self))
```
Comment (mprzypasniak99) [inline Frameworks/NetworkService/Sources/NetworkService/Logging/NetworkingLogger.swift:32]: Why is it commented out?

File: Handbid_iPad/Models/UserModel.swift [hunk 2]
```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2024 by Handbid. All rights reserved.
+
+import Arrow
+import NetworkService
+import SwiftData
```
Comment (mprzypasniak99) [inline Handbid_iPad/Models/UserModel.swift:5]: Why is SwiftData imported, when you're not using it here?

**Comment by mprzypasniak99** [comment]
Please explain why SwiftData wasn't used and what's the benefit of using custom built library/module for managing app data

**Review by mprzypasniak99 — COMMENTED** [review]
Please explain why SwiftData wasn't used and what's the benefit of using custom built library/module for managing app data

File: Frameworks/NetworkService/Sources/NetworkService/Logging/NetworkingLogger.swift (hunk 1 above)
Comment (BakJakub) [reply Frameworks/NetworkService/Sources/NetworkService/Logging/NetworkingLogger.swift:32]: You're right, unnecessarily commented out, while working on these shuffles I found a bug with maintaining a NetworkClient instance - while fixing this I forgot to uncomment it

File: Handbid_iPad/Data/Repositories/ChooseAuctionRepository.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2024 by Handbid. All rights reserved.
+
+import Arrow
+import Combine
+import Foundation
+import NetworkService
+
+protocol ChooseAuctionRepository {
+	// func fetchUserAuctionInventory(auctionId: Int) -> AnyPublisher<AuctionModel, Error>
```
[the remark this answers is not in the document]
Comment (BakJakub) [reply Handbid_iPad/Data/Repositories/ChooseAuctionRepository.swift:9]: You are right, this should not be here

File: Handbid_iPad/Models/UserModel.swift (hunk 2 above)
Comment (BakJakub) [reply Handbid_iPad/Models/UserModel.swift:5]: I forgot to delete this from the model

**Comment by BakJakub** [comment]
Today we will connect and talk about it

**Pull request closed without merging**